hail/python/hailtop/batch/backend.py

```diff
@@ -214,13 +213,29 @@ class ServiceBackend(Backend):
     >>> b.run() # doctest: +SKIP
     >>> service_backend.close()
 
+    If the Hail configuration parameter batch/billing_project was previously set
+    with ``hailctl config set``, then one may elide the billing_project
+    parameter.
+
+    >>> service_backend = ServiceBackend()
+    >>> b = Batch(backend=service_backend)
+    >>> b.run() # doctest: +SKIP
+    >>> service_backend.close()
+
     Parameters
     ----------
     billing_project: :obj:`str`
         Name of billing project to use.
     """
 
-    def __init__(self, billing_project):
+    def __init__(self, billing_project=None):
+        if billing_project is None:
+            billing_project = get_user_config().get('batch', 'billing_project')
+        if billing_project is None:
+            raise ValueError(
+                f'the billing_project parameter of ServiceBackend must be set '
+                f'or run hailctl config set batch/billing_project '
+                f'YOUR_BILLING_PROJECT')
```

Comment:
I don't like this split on multiple lines. Maybe put it in quotes. Hard to see it's one command.

Reply:
I added backticks. The user should see it all on one line.